cdc/owner/async_sink.go

```diff
@@ -34,10 +33,12 @@ const (
 	defaultErrChSize = 1024
 )
 
-// AsyncSink is an async sink design for owner
+// asyncSink is an async sink design for owner
 // The EmitCheckpointTs and EmitDDLEvent is asynchronous function for now
 // Other functions are still synchronization
-type AsyncSink interface {
+type asyncSink interface {
+	// Run the asyncSink
+	Run(ctx cdcContext.Context) error
```

Comment:
These methods also do not seem to be necessary to expose?

Reply:
In the codebase, the visibility of data items is quite chaotic. Maybe this needs to be addressed in a proper way.

Reply:
I think we can just keep the principle of minimal access. It will help us to find dead code.